src/ng/parse.js

```diff
@@ -1950,23 +1956,20 @@ function $ParseProvider() {
 
       var fn = parsedExpression.oneTime ? oneTimeInterceptedExpression : regularInterceptedExpression;
 
-      // Propogate the literal/oneTime attributes
+      // Propogate the literal/oneTime/constant attributes
       fn.literal = parsedExpression.literal;
       fn.oneTime = parsedExpression.oneTime;
+      fn.constant = parsedExpression.constant;
 
-      // Propagate or create inputs / $$watchDelegates
-      useInputs = !parsedExpression.inputs;
-      if (watchDelegate && watchDelegate !== inputsWatchDelegate) {
```

Comment:
Here's an example, I'll try to add a test for this...
```
$parse($interpolate("a{{b}}"), function(x) { return x + 'c'; })
```
`$interpolate` adds a special `$$watchDelegate` for an efficient way to watch the interpolated-expression, but that delegate of course knows nothing about the `x + 'c'` interceptor.  Previously this line we're commenting on would blindly copy the delegate creating an expression which produces the correct value when executed (with the `'c'` concatenated at the end) but the wrong value when watched due to the use of a `$$watchDelegate` that knows nothing about the `x + 'c'`...